_events/_0000-0000-communitymeeting-template.markdown

```diff
@@ -0,0 +1,26 @@
+---
+# rename this file to YYYY-MMDD-meeting-name.markdown - this will be the final URL
+# put your event date and time (24 hr) here:
+eventdate: 2023-02-28T15:00
+tz: UTC -8
+title: OpenSearch Community Meeting - 2023-02-28
+online: true
+signup:
+    url: https://www.meetup.com/opensearch/events/290444926/
+    title: Join on Meetup
+
+# below this tripple dash, describe your event. 
+# It should be desciptive, but as succinct as possible. Please include relevant agenda, or link to agenda for attendees to read. 
+
+# If this is a meeting/event driven by full-time members of the OpenSearch Project, and we intend to record, the disclaimer at the end is mandatory. If you have questions - reach out to Kris Freedain.
+---
+
+Join us for our online community meeting.
+
+Meeting topics and passcode are listed in the [collaborative agenda](https://forum.opensearch.org/t/opensearch-community-meeting-2023-0228/11895) on the forum.
+
+Feel free to comment on the agenda before the meeting if you want to add an item or have a question. Would you like to present? Reach out on the forumn thread - we welcome community presentations. 
+
+After the meeting, we will post the chat log and any meeting notes. We welcome you to keep the conversation going on the forum. 
+
+*By joining the OpenSearch Community Meeting, you grant OpenSearch, and our affiliates the right to record, film, photograph, and capture your voice and image during the OpenSearch Community Meeting (the “Recordings”). You grant to us an irrevocable, nonexclusive, perpetual, worldwide, royalty-free right and license to use, reproduce, modify, distribute, and translate, for any purpose, all or any part of the Recordings and Your Materials. For example, we may distribute Recordings or snippets of Recordings via our social media outlets.*
```

Comment:
coming from #1070: there are the wrong quotes again here around "Recordings". and there's no guarantee that they aren't also present in other files - unless you ran a global search & replace before adding this file here?
as suggested in the other ticket it might make sense to add a linter to the CI workflow for these things.